script/deploy

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env ruby
```

Comment:
I'm fine with you writing this file in ruby, but this means you're taking on responsibility for maintaining this file into the future. We have enough random python knowledge in the TA groups usually to do stuff with the toolkit, but I don't think there's much institutional Ruby knowledge.

Again, fine by me, just an FYI.

---

Looking at this script again, it's essentially just farming out to the docker commands, yeah? and the two PTY blocks are to pipe stdout/err from docker to the terminal? I might send you a Python version to look at.